### DISCUSSION TITLE
disconnected！！！！I can't find why is that

### DISCUSSION THREAD

**Discussion by vielonn** [Nimdy/Dedicated_Valheim_Server_Script#284] [General]
Is it the private address? Is there a god who can help me solve this problem。。。

║ Valheim Official Build: 8276154
║ Local Valheim Install:  8276154
║ Current Valheim Instance: ZHENGZHOUSHI
║ Server Name: dalangtaosha 
║ Internet:    Disconnected
║ External IP: 101.43.173.178 
║ Internal IP: 10.0.24.8  
║ Server Port: 2456 
║ Public Listing: Off
║ Server Status: active and running

**Comment by Nimdy** [comment]
Its really hard to identify the issues here.

I would very if your hosting company configurations.

Check and very if your ports are open:

https://geekstrom.de/valheim/check/

VALHEIMipaddress:VALHEIMport

Verify firewall configurations and allow ports through.

https://github.com/Nimdy/Dedicated_Valheim_Server_Script/wiki/How-to-Valheim-Server-Port-Configuration-DigitalOcean-AWS-Azure-and-GCP

The above link is examples of firewall rules for different hosting companies but the concept is the same no matter the hosting company.
